lambify/.github/workflows/deploy.yml

```diff
@@ -0,0 +1,31 @@
+name: Deploy
+on: workflow_dispatch
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Environment
+        run: |
+          echo "UID=$(id -u)" >> $GITHUB_ENV
+          echo "GID=$(id -g)" >> $GITHUB_ENV
+      - name: Cache
+        uses: actions/cache@v2
+        with:
+          path: |
+            ./vendor/bundle
+            ./node_modules
+          key: {{ "${{ runner.os }}-deploy-${{ hashFiles('Gemfile.lock', 'yarn.lock') }}" }}
```

Comment:
This is the caching part! A recent change to actions/cache now allows multiple paths and it does all the work. Because we got rid of the `.lamby` directory, the paths are not the same for both.